CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1172,90 +1267,102 @@ end
 
 -- Up arrow to open the inventory
 OpenInventoryButton = NewGui('ImageButton', 'OpenInventory')
-OpenInventoryButton.Size = UDim2.new(0, 44, 0, 40)
-OpenInventoryButton.Position = UDim2.new(0.5, -22, 1, -120)
-OpenInventoryButton.Image = "rbxasset://textures/ui/Backpack/ScrollUpArrow.png";
-OpenInventoryButton.MouseButton1Click:connect(function()
-	BackpackScript.OpenClose()
-end)
+do
```

Comment:
Partly for organization of logic, but also because I am creating a local variable `openInventoryButtonSelectionObject` in this block that we shouldn't be referencing later.